styles/forms/form.module.css

```diff
@@ -0,0 +1,189 @@
+.formContainer {
+  width: 350px;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.formInput {
+  width: 100%;
+  padding: 0.5rem;
+  margin-bottom: 0.5rem;
+}
+
+.formControl {
+  background: #E8EBED;
+  color: black;
+  width: 100%;
+  height: 42.5px;
```

Comment:
Avoid this kind of pixels, is better if is squire of 2

2, 4, 8, 16, 24, 32, 48 

42.5 is too specific won't escalate as good.